cfgov/cfgov/settings/base.py

```diff
@@ -794,3 +798,15 @@
         'TIMEOUT': None,
     }
 }
+
+# Set our CORS allowed origins based on a JSON list in the
+# CORS_ALLOWED_ORIGINS environment variable.
+try:
+    CORS_ALLOWED_ORIGINS = json.loads(
+        os.environ.get("CORS_ALLOWED_ORIGINS", "[]")
```

Comment:
could also use `os.getenv` but i think it essentially does the exact same thing.

Reply:
Yeah. We use `os.environ.get()` for most all of this.